apps/journeys-admin/src/components/JourneyVisitorsList/VisitorToolbar/VisitorToolbar.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Meta, Story } from '@storybook/react'
+import { useState, ComponentProps } from 'react'
+import IconButton from '@mui/material/IconButton'
+import FilterListRoundedIcon from '@mui/icons-material/FilterListRounded'
+import { journeysAdminConfig } from '../../../libs/storybook'
+import { VisitorToolbar } from './VisitorToolbar'
+
+const VisitorToolbarStory = {
+  ...journeysAdminConfig,
+  component: VisitorToolbar,
+  title: 'Journeys-Admin/JourneyVisitorsList/FilterDrawer/VisitorToolBar',
+  parameters: {
+    ...journeysAdminConfig.parameters,
+    layout: 'fullscreen'
+  }
+}
+
+const Template: Story<ComponentProps<typeof VisitorToolbar>> = (args) => {
+  const [open, setOpen] = useState(false)
+  return (
+    <>
+      <IconButton>
+        <FilterListRoundedIcon onClick={() => setOpen(!open)} />
+      </IconButton>
+    </>
```

Comment:
Shouldn't need to wrap this in another icon button
```suggestion
        <FilterListRoundedIcon onClick={() => setOpen(!open)} />
```